docs/source/ru/_toctree.yml

```diff
@@ -0,0 +1,980 @@
+- sections:
+  - local: index
+    title: 🤗 Transformers
+  - local: quicktour
```

Comment:
I would recommend commenting out or removing all the other entries here (besides `index.md`) since they don't exist yet and will cause the doc-builder to throw an error.

Reply:
Yes, I think this is great idea. I can only leave index.md for now while I'm translating other files. Just why I did it this way, I took a reference from transformers/docs/source/ms/ where there are also two files _techtree.xml and index.md and it's in the main branch right now. Anyway, I can delete [_toctree.yml] file for now